tests/test_iib_ops.py

```diff
@@ -213,6 +213,7 @@ def add_bundles_mock_calls_tester(
         binary_image="binary-image",
         overwrite_from_index=True,
         overwrite_from_index_token="overwrite_from_index_token",
+        deprecation_list=["bundle1"],
```

Comment:
Should I create a new function similar to `def add_bundles_mock_calls_tester` where I specify `deprecation_list` argument? I would do that but it seems ther is already a lot of duplication between `def add_bundles_mock_calls_tester` and `add_bundles_mock_calls_tester_not_called`

Reply:
I think you can enhance the existing test